arm-recoveryservicesbackup/2016-06-01/swagger/recoveryservicesbackup.json

```diff
@@ -0,0 +1,1548 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2016-06-01",
+    "title": "RecoveryServicesBackupClient",
+    "x-ms-code-generation-settings": {
+      "internalConstructors": true
+    }
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json",
+    "text/json",
+    "application/octet-stream"
+  ],
+  "produces": [
+    "application/json",
+    "text/json",
+    "application/octet-stream"
+  ],
+  "paths": {
+    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{resourceNamespace}/vaults/{resourceName}/backupFabrics/{fabricName}/protectionContainers/{containerName}": {
+      "get": {
+        "tags": [
+          "ProtectionContainers"
+        ],
+        "description": "Gets details of the specific container registered to your Recovery Services Vault.",
+        "operationId": "ProtectionContainers_BMSGetContainer",
+        "consumes": [],
+        "produces": [
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "name": "resourceName",
```

Comment:
For common parameter definitions like this, it's recommended to put it in the global parameters section (like you've done with `SubscriptionId`). Since AutoRest will create a property on the service client by default, you can customize this behavior with the [`x-ms-parameter-location` extension](https://github.com/Azure/autorest/blob/master/Documentation/swagger-extensions.md#x-ms-parameter-location) to make it remain a method parameter.

I realize this isn't in the [extensions documentation](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-extensions.md) for this repo, so I'll work to make sure it's put there. In general, we're looking to improve our documentation to make it easier to discover best practices like this. We'd like to make it more "scenario focused" and discoverable for service teams writing Swagger. Right now, you kinda need to know about all of the extensions to know which ones should be applied. Any feedback about our documentation is more than welcome!